source/includes/app.md

```diff
@@ -32,3 +32,29 @@ app.elements.destroy(button);
 ```
 
 The `elements` object can be used to query all scenes for an element, create elements, or destroy them.
+
+## SAI object
+
+```javascript
+// enables SAI
+app.sai.enabled = true;
+
+// focuses SAI on an element or the player
+app.sai.focus(button);
+app.sai.focus(button, onComplete);
+app.sai.unfocus();
+
+// displays some text next to SAI
+var message = "Hello, world.";
+app.sai.write(message);
+
+// speaks in plain text or SSML
+app.sai.speak(message);
+app.sai.speak(message, onComplete);
+
+// teleports SAI directly to an element or back to the player
```

Comment:
To match all our other docs, each of these sections should look like:

```
> Speaks in plain text or SSML

\```
api.sai.speak(message);
app.sai.speak(message, onComplete);
\```
```

Note the use of > and a newline after. See any of the other includes for an example.

Reply:
was following the existing style. will update the whole file.

Reply:
Ahhh, thanks